OSDOCS-14588: Updates to egress lockdown documentation (#93240)
openshift/openshift-docs: modules/rosa-hcp-set-environment-variables.adoc

// Module included in the following assemblies:
//
// * rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

ifeval::["{context}" == "rosa-hcp-egress-lockdown-install"]
:rosa-egress-lockdown-install:
endif::[]

:_mod-docs-content-type: PROCEDURE
[id="rosa-hcp-set-environment-variables_{context}"]

> = Set Environment Variables

Reviewer: ```suggestion
= Setting Environment Variables
```

Shouldn't this use a gerund?

.Procedure

. Set your environment variable by running the following command:
+
[source,terminal]
----
$ export <variable_name>=<variable_value>
----

. You can confirm that your variable has been set by running the following command:
+
[source,terminal]
----
$ echo <variable_name>
----
+
ifdef::rosa-egress-lockdown-install[]
.Suggested variables for disconnected {product-title} (ROSA) clusters
[cols="3",options="header"]
|===
|Variable name
|Variable value
|Notes

|`AWS_ACCOUNT_ID`
|`$(aws sts get-caller-identity --query Account --output text)`
|You must be logged in to your AWS account with `rosa login`.

|`CLUSTER_NAME`
|The name you want for your cluster.
|Your cluster name cannot exceed 26 characters.

|`OIDC_ID`
|The 32-digit ID for your OpenID Connect (OIDC) configuration.
|You generate this ID by running `rosa create oidc-config`.

|`OPERATOR_ROLES_PREFIX`
|The Operator role prefix.
|If you want to make your AWS account roles use the same prefix as your Operator roles, you can run `ACCOUNT_ROLES_PREFIX=$OPERATOR_ROLES_PREFIX` after setting your Operator role prefix variable.

|`PRIVATE_SUBNET`
|The ID of your private subnets.

> |This value should be enclosed in quotation marks (") and each subnet ID should be comma-separated.

Reviewer: ```suggestion
|You must enclose this value in quotation marks (") and separate the subnet IDs with commas.
```

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed.

|`REGION`
|Your AWS region.
| -

|`SUBNET_IDS`
|The IDs of all your subnets.

> |This value should be enclosed in quotation marks (") and each subnet ID should be comma-separated.

Reviewer: ```suggestion
|You must enclose this value in quotation marks (") and separate the subnet IDs with commas.
```

Same change

// Module included in the following assemblies:
//
// * rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

ifeval::["{context}" == "rosa-hcp-egress-lockdown-install"]
:rosa-disconnected-install:
endif::[]

:_mod-docs-content-type: PROCEDURE
[id="rosa-hcp-set-environment-variables_{context}"]

> == Set Environment Variables

Reviewer 2: ```suggestion
== Setting environment variables
```

> You may set following environment variables to streamline resource creation.

Reviewer 2: ```suggestion
Set the following environment variables to streamline resource creation.
```

.Procedure

. Set your environment variable by using the following command:
+
[source,terminal]
----
$ export <variable_name>=<variable_value>
----
+
ifdef::rosa-disconnected-install[]
.Suggested variables for disconnected {product-title} clusters
[cols="3",options="header"]
|===
|Variable name
|Variable value
|Notes

|`AWS_ACCOUNT_ID`
|$(aws sts get-caller-identity --query Account --output text)

> |You must be logged into your AWS account with `rosa login`.

Reviewer 2: ```suggestion
|You must be logged in to your AWS account with `rosa login`.
```

|`CLUSTER_NAME``

> |The name you want for your cluster

Reviewer 2: ```suggestion
|The name you want for your cluster.
```

|Your cluster name cannot exceed 26 characters.

|`OIDC_ID`
|The 32-digit ID for you OIDC configuration.
|You generate this ID by running `rosa create oidc-config`.

|`OPERATOR_ROLES_PREFIX`

> |Role prefix

Reviewer 2: ```suggestion
|The Operator role prefix.
```
